tests/pgautofailover_utils.py

```diff
@@ -121,6 +129,18 @@ def run_sql_query(self, query, *args):
             except psycopg2.ProgrammingError:
                 return None
 
+    def create_authenticated_users(self):
+        if self.authMethod is None:
```

Comment:
I would do it the other way around, it'd be less surprising:

``` python
    if self.authMethod:
        for authenticatedUser in ...:
```

Reply:
function is gone altogether